cmd/log-parser/agent.go

```diff
@@ -122,7 +122,22 @@ func unpackAgentLogEntry_v2(le LogEntry) (agent LogEntry, err error) {
 		case "container-id", "cid": // better to ensure consistency in the agent
 			agent.Container = v
 		case "level":
-			agent.Level = strings.ToLower(v)
+			switch v {
+			// match agent's slog short version log leveling to the common log levels
+			case "DEBG":
+				agent.Level = "debug"
+			case "ERRO":
+				agent.Level = "error"
+			case "INFO":
+				agent.Level = "info"
+			case "TRCE":
+				agent.Level = "trace"
+			case "WARN":
+				agent.Level = "warning"
+			default:
+				agent.Level = v
```

Comment:
@snir911 @littlejawa shouldn't "CRIT" be mapped to "critical", like the others ?

Reply:
@gkurz i wasn't sure about it, [logrus log levels](https://github.com/sirupsen/logrus#level-logging) aren't 1:1 matching to the slog levels, at first I thought CRIT may be translated to **Fatal** (or **Panic**?) but then decided to stick only with the more common levels. I'm open to other thoughts though..

Reply:
@snir911 There's code in the agent to translate from logrus to slog :

```
// Map logrus (https://godoc.org/github.com/sirupsen/logrus)
// log level to the equivalent slog log levels.
//
// Note: Logrus names are used for compatability with the previous
// golang-based agent.
#[instrument]
fn logrus_to_slog_level(logrus_level: &str) -> Result<slog::Level> {
    let level = match logrus_level {
        // Note: different semantics to logrus: log, but don't panic.
        "fatal" | "panic" => slog::Level::Critical,

        "critical" => slog::Level::Critical,
        "error" => slog::Level::Error,
        "warn" | "warning" => slog::Level::Warning,
        "info" => slog::Level::Info,
        "debug" => slog::Level::Debug,

        // Not in logrus
        "trace" => slog::Level::Trace,

        _ => bail!(ERR_INVALID_LOG_LEVEL),
    };

    Ok(level)
}
```
IIUC CRIT is slog's highest level and it doesn't panic, so I guess it should be fine to translate it to "critical".

Reply:
@gkurz that's odd, it doesn't seem that logrus has critical...
I'll add it